src/main/java/uk/gov/hmcts/reform/blobrouter/services/EnvelopeService.java

```diff
@@ -40,6 +40,11 @@ public EnvelopeService(
         return envelopeRepository.find(blobName, containerName);
     }
 
+    @Transactional(readOnly = true)
```

Comment:
Why is it transactional?

Reply:
see all other 'read' methods in this class

Reply:
Fine for this PR, but I wonder if that's needed.